.github/workflows/build-and-release.yml

```diff
@@ -6,10 +6,11 @@ on:
       - closed
     branches:
       - main
-
     paths-ignore:
       - ".github/**"
 
+  workflow_dispatch:
```

Comment:
Is this doing anything? Seems to be under `on` but doesn't have anything in it?

Reply:
woops... forgot to remove this. The purpose of this is to be able to run the workflow manually